database/transaction_test.go

```diff
@@ -79,7 +79,7 @@ func TestTxTable(t *testing.T) {
 
 		// Getting a table that doesn't exist should fail.
 		_, err = tx.GetTable("unknown")
-		require.EqualError(t, err, database.ErrTableNotFound.Error())
```

Comment:
If we want to check that a particular table does not exist, it’d be better to define the following type

```
type TableNotFoundError struct {
	Name string
}

func (*TableNotFoundError) Is(err error) bool {
	return err == ErrTableNotFound
}
```

Then unwrap the error:

```
var notFoundError TableNotFoundError
if errors.As(err, &notFoundError) {
	fmt.Println(notFoundError.Name)
}
```